docs/make.jl

```diff
@@ -1,10 +1,15 @@
+# Workaround for JuliaLang/julia/pull/28625
```

Comment:
Docker image now updated to use 1.0.1, could this be removed?

Reply:
Still needed for 0.7 but we can remove this since we only build the docs on Julia v1.0.1 now.

Reply:
I guess at this point we don't need CI for 0.7 any more, either, so perhaps you could disable it too in this PR?